build.gradle

```diff
@@ -21,9 +22,11 @@ subprojects {
     apply plugin: 'com.github.hierynomus.license'
 
     java {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
```

Comment:
I think it would be better to use the [`--release`](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.compile.CompileOptions.html#org.gradle.api.tasks.compile.CompileOptions:release) option.

Reply:
I kept it the same as before to avoid out of scope changes and keep it minimal (e.g. switching to release is not needed for the gradle 8.3 update).
On that topic I'd recommend moving to [toolchains](https://blog.gradle.org/java-toolchains) instead when pivoting away from these options.

Reply:
That would be my preference as well in general. We'll do it in a followup issue.

Reply:
@nlisker Are you okay with this PR as is then?

Reply:
Yes